### PR TITLE
[build] Install examples into DOC directory

### PR DESCRIPTION
**Pull request by justinjoy**
${prefix}/examples is too general location.

Btw, is this right repository for sending PR?

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -90,7 +90,7 @@ endif(ENABLE_TESTS)
 if(ENABLE_EXAMPLES)
     add_subdirectory(examples)
 endif(ENABLE_EXAMPLES)
-install(DIRECTORY examples DESTINATION .)
+install(DIRECTORY examples DESTINATION ${INSTALL_DOC_DIR})
```
Comment (loganek) [inline CMakeLists.txt]: Looks like `INSTALL_DOC_DIR` is not defined anywhere.

**Comment by loganek** [comment]
Hi @justinjoy Thanks a lot for the contribution. Yes, that's the right repository for sending the PR :)

**Comment by justinjoy** [comment]
Although `documents.cmake` defines same thing, I didn’t touch it and made a redundant if-else because I am not sure you want to have variables in CMakeLists on the top directory.

**Comment by loganek** [comment]
> Although `documents.cmake` defines same thing, I didn’t touch it and made a redundant if-else because I am not sure you want to have variables in CMakeLists on the top directory.

Hi @justinjoy I see your concersn about having one more global variable. However, I think it'd be still better to have a global variable, rather than repeating the code multiple times. What do you think?

**Comment by justinjoy** [comment]
I hope you are happy to have a new global variable. :)

**Comment by loganek** [comment]
@justinjoy Thank you very much for your contribution!

**Pull request merged**